appveyor.yml

```diff
@@ -1,47 +1,51 @@
 # Set up the build environment.
 environment:
-    # WORKAROUND START #
-    RUSTUP_USE_HYPER: 1
-    CARGO_HTTP_CHECK_REVOKE: false
-    # WORKAROUND END #
-
-    matrix:
-        - channel: stable
-          target: x86_64-pc-windows-msvc
-        - channel: beta
-          target: x86_64-pc-windows-msvc
-        - channel: nightly
-          target: x86_64-pc-windows-msvc
+  # WORKAROUND START #
+  RUSTUP_USE_HYPER: 1
+  CARGO_HTTP_CHECK_REVOKE: false
+  # WORKAROUND END #
 
+# Configure the build matrix.
+matrix:
+  - channel: stable
+    target: x86_64-pc-windows-msvc
+  - channel: beta
+    target: x86_64-pc-windows-msvc
+  - channel: nightly
+    target: x86_64-pc-windows-msvc
+
+# Allow for occasional compiler breakage on nightly Rust.
 matrix:
```

Comment:
Error parsing appveyor.yml: (Line: 18, Col: 1, Idx: 406) - (Line: 18, Col: 7, Idx: 412): Duplicate key

Is it possible the indentation on the two lines below this were necessary?  Alternatively I think we can just move `allow_failures` into the matrix key from above.